Parse/Internal/ReflectionHelpers.cs

```diff
@@ -7,7 +7,7 @@
 
 namespace Parse.Internal {
   static class ReflectionHelpers {
```

Comment:
Hmm. So this is because of `GetConstructors` shouldn't be used as extension, right?

If it is, can you add some description here on why is this not an extension for future reference?